hotel-management-app-v1/src/main/java/com/fb/hotelmanagementappv1/models/Room.java

```diff
@@ -0,0 +1,56 @@
+package com.fb.hotelmanagementappv1.models;
+
+public class Room {
+    private int id;
+    private boolean isOccupied;
+    private int roomSize;
+    private int numberOfBeds;
+    private boolean hasBalcony;
```

Comment:
What do you think about having a list of amenities? This will help us to expand it further without making any changes to the code structure. i.e. If we ever has to add to remove any new amenities